core-services/sync-rover-groups/_config.yaml

```diff
@@ -1,11 +1,19 @@
 cluster_groups:
+  build-farm:
```

Comment:
is this a spot that `cluster-init` now needs to be aware of? if so can we make a card to handle that?

Reply:
I will make a card for this.